consensus/model.h

```diff
@@ -18,10 +18,10 @@
 extern "C" {
 #endif
 
-typedef struct tips_pair {
+typedef struct hash_pair {
```

Comment:
we should give these fields different names then as well? left/right or one/two or whatever?

Reply:
sure